operator/docs/forwarding_logs_without_gateway.md

```diff
@@ -0,0 +1,112 @@
+
+* Deploy the Loki Operator and a `lokistack` instance for [OpenShift](./hack_loki_operator.md#hacking-on-loki-operator-on-openshift).
+
+* Deploy the OpenShift Logging Operator from the Operator Hub or using the following command locally:
+
+    ```console
+    make deploy-image deploy-catalog install
+    ```
+  
+* Create a Cluster Logging instance in the `openshift-logging` namespace with only `collection` defined.
+
+    For fluentd:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "fluentd"
+          fluentd: {}
+    ```
+
+    For vector:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "vector"
+          fluentd: {}
+    ```
+
+* By default, TLS is enabled on all components deployed by loki-operator. Because the service certificates are signed by a cluster-internal CA you need to set up a secret that enables the collector to validate the certificate returned by the distributor. The secret must exist in the openshift-logging namespace, and must have a key `ca-bundle.crt`.
+
+  The CA certificate is part of a ConfigMap that gets created by loki-operator as part of the LokiStack. Unfortunately this ConfigMap can not be used directly and has to be converted to a Secret readable by the collector.
+
+    Fetch the `ca-bundle.crt` using:
+
+    ```console
+    kubectl -n openshift-logging get cm lokistack-dev-gateway-ca-bundle -o jsonpath="{.data.service-ca\.crt}"
+    ```
+  
+* Once secret is fetched, create a new secret file:
+
+    ```console
+    kubectl -n openshift-logging create secret generic loki-distributor-ca \
+    --from-literal=ca-bundle.crt="<CA_BUNDLE>"
+    ```
+    
+    where `<CA_BUNDLE>` is the copied value from previous step.
+
+* Now create a ClusterLogForwarder CR to forward logs to LokiStack:
+
+  ```yaml
+  apiVersion: "logging.openshift.io/v1"
+  kind: "ClusterLogForwarder"
+  metadata:
+    name: "instance"
+    namespace: "openshift-logging"
+  spec:
+    outputs:
+     - name: loki-operator
+       type: "loki"
+       url: https://lokistack-dev-distributor-http.openshift-logging.svc:3100
+       secret:
+         name: loki-distributor-ca
+       loki:
+         tenantKey: log_type
+    pipelines:
+     - name: send-logs
+       inputRefs:
+       - application
+       - audit
+       - infrastructure
+       outputRefs:
+       - loki-operator
+  ```
+
+  _Note:_ The `tenantKey: log_type` gets resolved as `application`, `audit` or `infrastructure` by fluentd and vector based on the type of logs being collected. This is later used as different tenants for storing logs in Loki.
+
+## Troubleshooting
```

Comment:
Should we mention that `tenantKey` is currently broken when using a `fluentd` collector?

Reply:
I don't think we need to mention such things in doc. Also, that bug might be resolved by the time release happens.